src/SFML/Graphics/Font.cpp

```diff
@@ -468,6 +468,26 @@ const Texture& Font::getTexture(unsigned int characterSize) const
     return m_pages[characterSize].texture;
 }
 
+////////////////////////////////////////////////////////////
+void Font::setSmooth( bool smooth )
+{
+    if (smooth != m_isSmooth)
+    {
+        m_isSmooth = smooth;
+
+        for (int i = 0; i < m_pages.size(); ++i)
```

Comment:
Should use the key type `unsigned int` or probably even better, iterate over the container with iterators.

Reply:
Done, with iterators